app/src/main/java/com/nutomic/syncthingandroid/service/NotificationHandler.java

```diff
@@ -57,7 +57,7 @@ public NotificationHandler(Context context) {
             mPersistentChannel.enableLights(false);
             mPersistentChannel.enableVibration(false);
             mPersistentChannel.setSound(null, null);
-            mPersistentChannel.setShowBadge(false); // Disables the dot next to the app icon
```

Comment:
I've removed the comment as it doesn't exist in STF and "setShowBadge" is pretty clear what it does from my point of view.